src/ffi.rs

```diff
@@ -19,40 +19,20 @@ impl NewtonPoint {
 }
 
 #[no_mangle]
-pub extern "C" fn newton_new_field(
-    g: f32,
-    solar_mass: f32,
-    min_dist: f32,
-    max_dist: f32,
-) -> *mut Field {
-    let field = Field::new(g, solar_mass, min_dist, max_dist);
-    let boxed = Box::new(field);
+pub extern "C" fn newton_new_environment() -> *mut Environment {
```

Comment:
what does this mean
`*mut Environment` ? 
does this mean that it will return an Referenz of Object of type Environment ?

Reply:
Yes, a mutable reference.